spec/blackjack/player_spec.rb

```diff
@@ -75,17 +75,17 @@
   end
 
   # Need refactoring to make the mock work
-  # describe "make_move" do
-  #   it "hits if the player types h" do
-  #     @player.make_move
-  #     expect(@player.hand.count).to eq 1
-  #   end
-  #
-  #   it "stands if the player types s" do
-  #     # mock (test double, fake implementation of gets)
-  #     allow(self).to receive(:gets) {"s"}
-  #     @player.make_move
-  #     expect(@player.stand_status).to be true
-  #   end
-  # end
+  describe "make_move" do
+    it "hits if the player types h" do
+      @player.make_move
+      expect(@player.hand.count).to eq 1
+    end
+
+    it "stands if the player types s" do
+      # mock (test double, fake implementation of gets)
+      allow(Kernel).to receive(:gets) {"s"}
```

Comment:
I was right about using `Kernel` here, but we couldn't get it to work because of the incorrect while loop we wrote.